README.md

```diff
@@ -32,3 +33,7 @@ figure out a remediation timeline.
 
 We aim for initial response to vulnerability reports within 48
 hours. The timeline for fixes depends on the complexity of the issue.
+
+
+## License
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fmetallb%2Fmetallb.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fmetallb%2Fmetallb?ref=badge_large)
```

Comment:
This seems like a bit much ... maybe we can remove this and leave the status badge above

Reply:
Yep, also it's not clear how this reports issues while the CI passed successfully.

Reply:
Agree, not sure how to change this though. Guess we can't ask to fossabot

Reply:
we can edit the branch ourselves

> Add more commits by pushing to the add-license-scan-badge branch on fossabot/metallb.

Reply:
removed